include/pstore/adt/chunked_vector.hpp

```diff
@@ -51,10 +51,23 @@
 
 namespace pstore {
 
+    /// Chunked-vector is a sequence-container which uses a list of large blocks ("chunks") to
+    /// ensure very fast append times at the expense of only permitting bi-directional iterators:
+    /// random access is not supported, unlike std::deque<> or std::vector<>.
+    ///
+    /// \tparam T The type of the elements.
+    /// \tparam ElementsPerChunk The number of elements in an individual chunk.
+    /// \tparam ActualSize The storage allocated to an individual element. Normally equal to
+    ///   sizeof(T), this can be increased to allow for dynamically-sized types.
+    /// \tparam ActualAlign The alignment of an individual element. Normally equal to alignof(T).
     template <typename T,
-              std::size_t ElementsPerChunk = std::max (4096 / sizeof (T), std::size_t{1})>
+              std::size_t ElementsPerChunk = std::max (4096 / sizeof (T), std::size_t{1}),
```

Comment:
> Should 4096 be declared as constant (with a meaningful name)?

Personally I don’t think that it’s necessary. `ElementsPerChunk` gives the constant a meaningful name. What sort of name did you have in mind?

> Should `std::size_t{1}` change to `std::size_t{1U}`

No, because I’m defining a `size_t` constant, not an `unsigned`.

Reply:
> 
> 
> > Should 4096 be declared as constant (with a meaningful name)?
> 
> Personally I don’t think that it’s necessary. `ElementsPerChunk` gives the constant a meaningful name. What sort of name did you have in mind?
>

Yes, you are right. I can't think other name.

> 
> > Should `std::size_t{1}` change to `std::size_t{1U}`
> 
> No, because I’m defining a `size_t` constant, not an `unsigned`.

I see, thanks.